src/Provider/Provider.Web/Controllers/VacanciesController.cs

```diff
@@ -38,5 +39,23 @@ public async Task<IActionResult> Vacancies([FromQuery] string filter, [FromQuery
 
             return View(vm);
         }
+
+        private void SetFilters(string filter, string search)
```

Comment:
`SetFiltersCookie`

Reply:
SaveFiltersInCookie to match TryGetFiltersFromCookie